credentials/customer-loyalty/credential.json

```
@@ -0,0 +1,25 @@
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://contexts.vcplayground.org/examples/customer-loyalty/v1.json"
+  ],
+  "type": ["VerifiableCredential", "CustomerLoyaltyCredential"],
+  "description": "Earn Big Rewards at Big Retail Store by presenting your loyalty card!",
+  "image": "https://playground.chapi.io/examples/customer-loyalty/image.png",
+  "name": "Big Retail Customer Loyalty Card",
+  "credentialSubject": {
+    "id": "did:key:123",
+    "customerLoyaltyCard": {
+      "type": "CustomerLoyaltyCard",
+      "branchCode": "461",
+      "identifier": "9712323901"
+    }
+  },
+  "issuanceDate": "2023-06-12T00:00:00Z",
+  "issuer": {
+    "id": "did:key:z6MkrHKzgsahxBLyNAbLQyB1pcWNYC9GmywiWPgkrvntAZcj",
+    "image": "https://playground.chapi.io/examples/retail-coupon/logo.png",
```

Use vcplayground.org:
```suggestion
    "image": "https://vcplayground.org/examples/retail-coupon/logo.png",
```
I'd consider copying the logo image over to customer-loyalty so the example is self contained and resistant to changes in the retail-coupon example.